manifests/params.pp

```diff
@@ -23,6 +23,7 @@
   $manage_download_dir = false
   $download_filename   = 'vault.zip'
 
+  $manage_config_file  = true
```

Comment:
Can this get moved up to be declared alongside the other `config_*` parameters?

Reply:
Moved